src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -261,7 +261,7 @@ public bool NotUsingHttps(string url)
                     waitingTasks.Add(requester.ExecuteAsync(new WaitForRaftIndexCommand(index), context));
                 }
 
-                await Task.WhenAll(waitingTasks);
+                await Task.WhenAny(waitingTasks);
```

Comment:
This code is now running _while the request is running_, this is probably not a good idea.